### PR TITLE
Prevent the first level load duplication

### PR DESCRIPTION
**Pull request by aefimov3**
## What does this PR do?

Enhance duplicate command check to avoid initial level loaded twice

## How was this PR tested?

Local test on Windows, iOS, Android (mobile unaffected).

### CODE REVIEW

File: Code/Framework/AzCore/AzCore/Console/Console.cpp [hunk 1]
```diff
@@ -120,29 +120,32 @@ namespace AZ
...
 
-#ifdef CARBONATED
-            // A protection against an execution of 2 the same commands one by one.
-            // For example, this happens because the "LoadLevel <path>" command from "Registry\autoexec.game.setreg" is compiled into
-            // "Cache\pc\bootstrap.game.<profile>.setreg" as a sequence of the same commands.
-            // This fix compares the new deffered command with the last one in the list and ignores it if they are fully equal
+#if defined(CARBONATED) && !(defined(AZ_PLATFORM_IOS) || defined(AZ_PLATFORM_ANDROID))
+            // A protection against an execution of the same command more than once.
+            // If LoadLevel command is in Registry\autoexec.game.setreg then it is also copied to
+            // "Cache\pc\bootstrap.game.<profile>.setreg". The engine executes both setrget files for Windows standalone client.
```
Comment (alexeykostin) [inline Code/Framework/AzCore/AzCore/Console/Console.cpp]: setrget  setreg

File: Code/Framework/AzCore/AzCore/Console/Console.cpp [hunk 2]
```diff
@@ -120,29 +120,32 @@ namespace AZ
...
+            // The issue happens if there is a source setreg file available, which is not actual for mobile platforms.
+            // This code piece below compares the new deffered command with all the queue commands, it an exact match found then it ignores the new command.
```
Comment (alexeykostin) [inline Code/Framework/AzCore/AzCore/Console/Console.cpp]: deffered -> deferred

it an -> if an

**Review by vmednonogov — APPROVED** [review]
LGTM

File: Gems/VideoPlaybackFramework/Code/Include/VideoPlaybackFramework/VideoPlaybackBus.h [hunk 3]
```diff
@@ -52,7 +52,7 @@ namespace VideoPlaybackFramework
         /*
         * Clear resources and hide the video, this works for mobile native player only
         */
-        virtual void ClearResourcesAndHide() {}
+        virtual void ClearResourcesAndHide() = 0;
```
Comment (alexeykostin) [inline Gems/VideoPlaybackFramework/Code/Include/VideoPlaybackFramework/VideoPlaybackBus.h:55]: Is it needed for this PR?

File: Code/Framework/AzCore/AzCore/Console/Console.cpp [hunk 4]
```diff
@@ -120,29 +120,32 @@ namespace AZ
...
             if (m_deferredCommands.size() > 0)
             {
-                DeferredCommand& lastCommand = m_deferredCommands.back();
-                if (lastCommand.m_command == deferredCommand.m_command &&
-                    lastCommand.m_arguments.size() == deferredCommand.m_arguments.size())
+                for (DeferredCommand& checkCommand : m_deferredCommands)
```
Comment (alexeykostin) [inline Code/Framework/AzCore/AzCore/Console/Console.cpp]: Can it be `const DeferredCommand& checkCommand` ?

**Review by carb-lloyd — APPROVED** [review]

File: Gems/VideoPlaybackFramework/Code/Include/VideoPlaybackFramework/VideoPlaybackBus.h (hunk 3 above)
Comment (aefimov3) [reply Gems/VideoPlaybackFramework/Code/Include/VideoPlaybackFramework/VideoPlaybackBus.h:55]: It is related to video playback feature, the one I merged to game's main.

**Review by alexeykostin — APPROVED** [review]

**Pull request merged**